src/Umbraco.Web.UI.Client/src/views/content/overlays/publish.controller.js

```diff
@@ -12,6 +12,8 @@
         vm.dirtyVariantFilter = dirtyVariantFilter;
         vm.pristineVariantFilter = pristineVariantFilter;
 
+        $scope.getVariantDisplayName = variantHelper.getDisplayName;
```

Comment:
This controller is configured to use `vm` so this should be assigned to vm not $scope.